### ISSUE TITLE
Add RSS for new content added

### ISSUE THREAD

**Issue by lwhsu** [freebsd/freebsd-papers#72]
It is nice to have a RSS for new content get added, but we probably shouldn't just go with git commit and merge because it's better to skip the minor fixes and site updates.

**Comment by rwp0** [comment]
Yes, unfortunately I see no RSS for https://papers.freebsd.org/ to add on my reader.
